voidcrew/mapping/_mapping.dm

```diff
@@ -41,4 +51,34 @@
 			if(SYNDICATE_SHIP)
 				syn_ship_list[initial(shuttles.name)] = shuttles
 
+/datum/controller/subsystem/mapping/get_station_center()
+	return SSovermap.overmap_centre || locate(OVERMAP_LEFT_SIDE_COORD, OVERMAP_NORTH_SIDE_COORD, OVERMAP_Z_LEVEL)
+
+/datum/controller/subsystem/mapping/get_turf_above(turf/T)
+	return SSovermap.calculate_turf_above(T)
+
+/datum/controller/subsystem/mapping/get_turf_below(turf/T)
+	return SSovermap.calculate_turf_below(T)
+
+/get_safe_random_station_turf(list/areas_to_pick_from)
```

Comment:
```suggestion
/proc/get_safe_random_station_turf(list/areas_to_pick_from)
```
What da hell

Reply:
I'm overriding already existing global procs

Reply:
god i hate how it looks